### PR TITLE
[OSCD] LOLBAS CL_Mutexverifiers - powershell

### PR DESCRIPTION
**Pull request by nsaddler**
Adding powershell_CL_Mutexverifiers_LOLScript.yml Rule (#1014 - 107)

### CODE REVIEW

File: rules/windows/powershell/powershell_CL_Mutexverifiers_LOLScript.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    selection2:
+        EventID: 4104
+        ScriptBlockText|contains:
+          - 'CL_Mutexverifiers.ps1'
+          - 'runAfterCancelProcess'
+    timeframe: 1m
+    condition: 
+        - selection
+      # PS > Import-Module c:\Windows\diagnostics\system\Audio\CL_Mutexverifiers.ps1; runAfterCancelProcess c:\Evil.exe
+        - selection2 | count(ScriptBlockText) by Computer > 2
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_CL_Mutexverifiers_LOLScript.yml:31]: It seems that it's very similar to the https://github.com/Neo23x0/sigma/pull/1184
Could you please adjust it?

Comment (yugoslavskiy) [reply rules/windows/powershell/powershell_CL_Mutexverifiers_LOLScript.yml:31]: Hello @nsaddler! Would you like to proceed with working on this rule?

Comment (nsaddler) [reply rules/windows/powershell/powershell_CL_Mutexverifiers_LOLScript.yml]: Sorry for delay!
Of course I would, I did the same as in the #1184 (splitting into two rules). I think these conditions are equivalent, an adversary can enter it as by one line so by two ones.

Comment (yugoslavskiy) [reply rules/windows/powershell/powershell_CL_Mutexverifiers_LOLScript.yml]: Thanks!

**Comment by yugoslavskiy** [comment]
Thank you very much! Great job, @nsaddler!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**